src/Microsoft.Data.Sqlite/SqliteEngine.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Data.Sqlite.Interop;
+
+namespace Microsoft.Data.Sqlite
+{
+    public static class SqliteEngine
```

Comment:
Also note, you can't change it after it's loaded. If you don't want to use it, don't call it. Having another API to un-call it is silly.